2021-09-20 23:50:48,057 INFO    Thread-18 :567320 [wandb_setup.py:_flush():69] setting env: {'project': 'simple-classification-sweep-2', 'entity': 'sang-hyun', 'root_dir': '/tf/CI/MyTorchTemplate', 'run_id': '6ezzz1ow', 'sweep_param_path': '/tf/CI/MyTorchTemplate/wandb/sweep-o9tso7nr/config-6ezzz1ow.yaml', 'sweep_id': 'o9tso7nr'}
2021-09-20 23:50:48,057 INFO    Thread-18 :567320 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-20 23:50:48,057 INFO    Thread-18 :567320 [wandb_init.py:_log_setup():342] Logging user logs to /tf/CI/MyTorchTemplate/wandb/run-20210920_235048-6ezzz1ow/logs/debug.log
2021-09-20 23:50:48,057 INFO    Thread-18 :567320 [wandb_init.py:_log_setup():343] Logging internal logs to /tf/CI/MyTorchTemplate/wandb/run-20210920_235048-6ezzz1ow/logs/debug-internal.log
2021-09-20 23:50:48,057 INFO    Thread-18 :567320 [wandb_init.py:init():375] calling init triggers
2021-09-20 23:50:48,057 INFO    Thread-18 :567320 [wandb_init.py:init():382] wandb.init called with sweep_config: {'ARCHITECTURE': 'single-layer-perceptron', 'BATCH_SIZE': 64, 'DATASET': 'MNIST-handwrite', 'DATA_PATH': '../dataset/', 'EPOCHS': 10, 'LEARNING_RATE': 0.03850640050041727, 'NOW': 'None', 'NUM_WORKERS': 1, 'SAVEFIG_NAME': 'result.png'}
config: {}
2021-09-20 23:50:48,057 INFO    Thread-18 :567320 [wandb_init.py:init():424] starting backend
2021-09-20 23:50:48,057 INFO    Thread-18 :567320 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-20 23:50:48,061 INFO    Thread-18 :567320 [backend.py:ensure_launched():135] starting backend process...
2021-09-20 23:50:48,081 INFO    Thread-18 :567320 [backend.py:ensure_launched():140] started backend process with pid: 570433
2021-09-20 23:50:48,082 INFO    Thread-18 :567320 [wandb_init.py:init():429] backend started and connected
2021-09-20 23:50:48,083 INFO    Thread-18 :567320 [wandb_run.py:_config_callback():843] config_cb None None {'ARCHITECTURE': 'single-layer-perceptron', 'BATCH_SIZE': 64, 'DATASET': 'MNIST-handwrite', 'DATA_PATH': '../dataset/', 'EPOCHS': 10, 'LEARNING_RATE': 0.03850640050041727, 'NOW': 'None', 'NUM_WORKERS': 1, 'SAVEFIG_NAME': 'result.png'}
2021-09-20 23:50:48,083 INFO    Thread-18 :567320 [wandb_init.py:init():477] updated telemetry
2021-09-20 23:50:48,083 INFO    Thread-18 :567320 [wandb_init.py:init():500] communicating current version
2021-09-20 23:50:48,590 INFO    Thread-18 :567320 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-20 23:50:48,590 INFO    Thread-18 :567320 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-20 23:50:48,829 INFO    Thread-18 :567320 [wandb_init.py:init():540] starting run threads in backend
2021-09-20 23:50:49,706 INFO    Thread-18 :567320 [wandb_run.py:_console_start():1601] atexit reg
2021-09-20 23:50:49,707 INFO    Thread-18 :567320 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-09-20 23:50:49,707 INFO    Thread-18 :567320 [wandb_run.py:_redirect():1480] Redirecting console.
2021-09-20 23:50:49,707 ERROR   Thread-18 :567320 [wandb_run.py:_redirect():1539] Failed to redirect.
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/wandb_run.py", line 1532, in _redirect
    out_redir.install()
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/usr/lib/python3.6/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2021-09-20 23:50:49,707 INFO    Thread-18 :567320 [wandb_init.py:init():565] run started, returning control to user process
2021-09-20 23:51:38,788 INFO    Thread-18 :567320 [wandb_run.py:finish():1271] finishing run sang-hyun/simple-classification-sweep-2/6ezzz1ow
